### ISSUE TITLE
the order of importing TalyorSeries should not matter

### ISSUE THREAD

**Issue by JeffreySarnoff** [JuliaDiff/TaylorSeries.jl#78]
[the second package to find order dependance](https://github.com/MikeInnes/MacroTools.jl/issues/22)

I had the same problem many months ago.  Seeing Mike's note allowed me to inform you of that.

**Comment by lbenet** [comment]
Thanks for the note Jeffrey! Can you please be more specific about Mike's notes? I guess you refer to @MikeInnes, don't you?

**Comment by JeffreySarnoff** [comment]
Mike is @MikeInnes, instead of "note" read "issue" -- there is no other relevant note.

**Comment by fhenneke** [comment]
I have a similar problem with TaylorSeries and ForwardDiff.
```julia
using TaylorSeries, ForwardDiff
a = ForwardDiff.Dual(1.)
a * a
```
works as intended, whereas
```julia
using ForwardDiff, TaylorSeries
a = ForwardDiff.Dual(1.)
a * a
```
produces an error.
```julia
ERROR: * not defined for ForwardDiff.Dual{0,Float64}
 in *(::ForwardDiff.Dual{0,Float64}, ::ForwardDiff.Dual{0,Float64}) at ./promotion.jl:256
```

**Comment by lbenet** [comment]
Thanks for reporting. I truly do not know what's going on here.

**Comment by JeffreySarnoff** [comment]
@lbenet ask the dev-list?

**Comment by lbenet** [comment]
Closing since it is (somehow) fixed in #95.
